denizen_scripts/hub1/repo-link/Adriftus_Moderator_Panel/panels/mod_send_inv.dsc

```diff
@@ -0,0 +1,41 @@
+# -- SEND PANEL
+mod_send_inv:
+  type: inventory
+  debug: false
+  title: <&6>A<&e>MP <&f>› <&5>Send <&e><player.flag[amp_map].as_map.get[uuid].as_player.name> <&5>to Server.
+  inventory: CHEST
+  size: 27
+  definitions:
+    x: <item[air]>
+    b1: <item[magenta_stained_glass_pane].with[display_name=<&r>]>
+    b2: <item[purple_stained_glass_pane].with[display_name=<&r>]>
+    back: <item[red_stained_glass_pane].with[display_name=<&c><&l>↩<&sp>Actions<&sp>panel;nbt=<list[to/actions]>]>
+    head: <item[mod_player_item]>
+  procedural items:
+    - define inventory:<list>
+    - foreach <yaml[bungee.config].list_keys[servers]> as:server:
+      - if <yaml[bungee.config].read[servers.<[server]>.show_in_play_menu]>:
+        - define name <[server]>
+        - define lore <list.include[<yaml[bungee.config].parsed_key[servers.<[server]>.description]>]>
+        - define lore:|:<&d>Right<&sp>Click<&sp>to<&sp>transfer<&co>
+        - define lore:|:<player.flag[amp_map].as_map.get[uuid].as_player.name>
+        - define lore:|:<&d>Clic<&sp>Droit<&sp>pour<&sp>envoyer<&co>
+        - define nbt <list[SERVER/<[server]>]>
+        - define item <yaml[bungee.config].read[servers.<[server]>.material].as_item.with[display_name=<[name]>;lore=<[lore]>]>
+        - define inventory:|:<[item]>
```

Comment:
This value being inserted into the list is a single list value, `:->:` is preferred for single entry insertions over `:|:`